hack/update-rhcos-bootimage.py

```diff
@@ -23,6 +23,8 @@
     }
     for entry in meta['amis']
 }
+newmeta['azure'] = meta['azure']
```

Comment:
this can be part of https://github.com/openshift/installer/pull/2089/files#diff-ef97fb21d4164b6fdb1e5a144d91e4e7R17

Reply:
right?

Reply:
Indeed, it can. Fixed up and pushed.